packages/fuselage/src/components/Table/TableSelection/TableSelectionButton.tsx

```diff
@@ -1,11 +1,11 @@
 import type { ComponentProps } from 'react';
 import React from 'react';
 
-import type Box from '../Box';
-import { Button } from '../Button';
+import type Box from '../../Box';
+import { Button } from '../../Button';
 
 type TableSelectionButtonProps = ComponentProps<typeof Box>;
```

Comment:
shouldn't it be `ComponentProps<typeof Button>`?

Reply:
it should